frameworks/JavaScript/graphql-express-mysql/src/server.js

```diff
@@ -0,0 +1,316 @@
+        }
+
+        const newFortune = { "id": 0, "message": "Additional fortune added at request time." };
+        toRet.push(newFortune);
+        toRet.sort((a, b) => (a.message < b.message) ? -1 : 1);
+
+        const htmlToRet = await spoofHTML(toRet);
+        const sanitizedHtml = escape(htmlToRet);
+
+        res.contentType('html');
+        res.setHeader('Content-Length', sanitizedHtml.length + 32);
+        res.status(200);
+
+        res.real_end(sanitizedHtml);
+    }
+};
+
+const spoofHTML = arr => {
+
+    return new Promise((resolve, reject) => {
+
+        var count = 0;
+
+        var htmlToRet = `<!DOCTYPE html>
+        <head>
+            <title>Fortunes</title>
+        </head>
+        <body>
+        <table>
+        <tr>
+        <th>
+            id
+        </th>
+        <th>
+            message
+        </th>
+        </tr>
+        `;
+
+        for (let fortune of arr) {
+
+            htmlToRet += `
+            <tr>
+                <td>
+                ${fortune.id}
+                </td>
+                <td>
+                ${fortune.message}
+                </td>
+            </tr>
+        `
+            count++;
+        }
+
+        htmlToRet += `
+        </table>
+        </body>
+    </html>`;
+    
+        if(count == arr.length)resolve(htmlToRet);
+
+    });
+
+};
+
+const multipleJsonFormatResponseFromWorld = (res) => {
+
+    res.real_end = res.end;
+
+    res.end = (data) => {
+        var toRet;
+        const json = JSON.parse(data.toString('utf8'));
+
+        if(json.data.multipleDatabaseQueries) {
+            toRet = json.data.multipleDatabaseQueries;
+        } else {
+            toRet = [];
+        }
+
+        const jsonToRet = JSON.stringify(toRet);
+        
+        setResponseHeaders(res, jsonToRet.length);
+
+        res.real_end(jsonToRet);
+    }
+};
+
+const singleJsonFormatResponseFromWorld = (res, queryName) => {
+
+    res.real_end = res.end;
+    
+    res.end = (data) => {
+        var to_ret;
+        const json = JSON.parse(data.toString('utf8'));
+
+        console.log('hola', json);
+
+        if(json.data[`${queryName}`]) {
+            to_ret = {
+                id: json.data[`${queryName}`].id,
+                randomNumber: json.data[`${queryName}`].randomNumber
+            };
+        } else {
+            to_ret = {id: null, randomNumber: null};
+        }
+        const jsonToRet = JSON.stringify(to_ret);
+        
+        setResponseHeaders(res, jsonToRet.length);
+
+        res.real_end(jsonToRet);
+    };
+};
+
+const setResponseHeaders = (res, jsonLength) => {
+
+    var now = new Date();
+
+    res.status(200);
+    res.contentType('application/json', 'charset=UTF-8');
+    res.setHeader('date', dateFormat(now, "yyyy-mm-dd hh:MM:ss o"));
+    res.setHeader('Server', 'GraphQL-Express-MySQL');
+    res.setHeader('Content-Length', jsonLength);
+};
+
+app.listen(port, () => {
+    console.log(`Listening on localhost:${port}`);
+});
```

Comment:
This code looks fine to me, but I think that this **should** be another permutation in the `express` folder (since this is express most of the way, and we're just treating GraphQL as a complicated ORM).